initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -937,6 +936,20 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="maxPersistentIdleTime" type="xs:integer" use="optional" default="60">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>The maximum idle time (-1 for unlimited, 0 to disable).</tooltip>
```

Comment:
Time (milliseconds) to wait for additional requests on a persistent http connection. (-1 for unlimited, 0 to disable).

Reply:
Would (ms) suffice? @JamesDeFabia

Reply:
@JamesDeFabia @kenrowland Here its number of seconds, not milli-seconds

Reply:
@mayx In Roxie.xsd.in, the attribute uses milliseconds. Should we be consistent?

Reply:
How about this: The maximum idle time in seconds (-1 for unlimited, 0 to disable).

I don't see a maxPersistentIdleTime in roxie, perhaps I am missing the equivalent?

Reply:
maxHttpKeepAliveWait in roxie.xsd.in 
https://github.com/hpcc-systems/HPCC-Platform/blob/master/initfiles/componentfiles/configxml/roxie.xsd.in

Reply:
@JamesDeFabia you can raise another issue if you think milli-seconds makes more sense. But right now the code in ESP (already merged in) is in seconds and the configuration should conform to that. (by the way this part of ESP code got in before Roxie :) @kenrowland please keep it as seconds to conform to ESP code.